nginx.conf

```diff
@@ -0,0 +1,12 @@
+# this is a nginx based configuration for the dev proxy
+server {
+    listen 3128;
+    location / {
+      access_log off;
+      proxy_pass http://g4j-localstack:4566/restapis/gateway/local/_user_request_/;
+#       proxy_set_header X-Real-IP $remote_addr;
```

Comment:
Do we need these or can we deleted the stuff that's commented out?

Reply:
yeah, can probably remove that :)